index.html

```diff
@@ -14,18 +14,19 @@
 
     <div id="checking" class="account">
       <h2>Checking</h2>
-      <div class="balance">$0</div>
-      <input class="input" type="text" placeholder="enter an amount" />
-      <input class="deposit" type="button" value="Deposit" />
-      <input class="withdraw" type="button" value="Withdraw" />
+      <div id="checkingBalance" class"balance">$0</div>
+      <input id="checkingAmount" type="text" placeholder="enter an amount" />
+      <input id="checkingDeposit" type="button" value="Deposit" />
+      <input id="withdrawDepositt" type="button" value="Withdraw" />
     </div>
 
     <div id="savings" class="account">
       <h2>Savings</h2>
-      <div class="balance">$0</div>
-      <input class="input" type="text" placeholder="enter an amount" />
-      <input class="deposit" type="button" value="Deposit" />
-      <input class="withdraw" type="button" value="Withdraw" />
+      <div id="savingBalance">$0</div>
+      <input id="savingAmount" type="text" placeholder="enter an amount" />
+      <input id="savingDepositt" type="button" value="Deposit" />
+      <input id="withdrawDeposit" type="button" value="Withdraw" />
```

Comment:
For this assignment, you had to just write javascript but not change the html. There will often be times as a developer that you cannot change the html that your javascript is interfacing with.

The key here is how to select elements (write selectors) and use them to select elements in jQuery like: `$(".balance .input")`